### PR TITLE
bump deps and fix rubocop

### PR DESCRIPTION
**Pull request by andyborn**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
       - name: Set up Google Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@v0
```
Comment (dim) [inline .github/workflows/test.yml]: why not v1? https://github.com/google-github-actions/setup-gcloud#service-account-key-json

**Comment by dim** [comment]
@andyborn looks like CI is failing

**Comment by dim** [comment]
@andyborn PING!

**Comment by andyborn** [comment]
> @andyborn PING!

Sorry, yes I got very stuck previously trying to fix this one.  will take another look this week.

**Comment by andyborn** [comment]
> @andyborn looks like CI is failing

@dim so re-running each ruby version stage individually and eventually all pass.  failures were happening on the FTP specs.

File: ftp/bfs-ftp.gemspec [hunk 2]
```diff
@@ -13,12 +13,11 @@ Gem::Specification.new do |s|
 
   s.executables   = []
   s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- spec/*`.split("\n")
   s.require_paths = ['lib']
   s.required_ruby_version = '>= 2.7'
 
   s.add_dependency 'bfs', s.version
-  s.add_dependency 'net-ftp', '>= 0.1.3'
+  s.add_dependency 'net-ftp', '< 0.2'
```
Comment (dim) [inline ftp/bfs-ftp.gemspec:20]: separate PR? looking at the code, not that much has changed, tbh

**Review by dim — APPROVED** [review]

**Pull request merged**